datumaro/components/visualizer.py

```diff
@@ -77,6 +79,7 @@ class Visualizer:
     def __init__(
         self,
         dataset: IDataset,
+        draw_only_image: bool = False,
```

Comment:
This could be achieved by setting alpha to zero. Of course, by providing this argument, we don't need to get access to the annotation data internally. But I am just concerning this occurs the confusion during external uses of visualization.

Reply:
Removed this flag and replaced it with `alpha=0` for this functional (no annotation drawing).